pkg/labels/matcher.go

```diff
@@ -115,15 +116,22 @@ func (m *Matcher) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
+	if v1m.IsEqual == nil {
```

Comment:
Can we instead of the pointer, do :

```
var v1m apiV1Matcher {
  isEqual: true
}
f err := json.Unmarshal(data, &v1m); err != nil {
```

That would probably solve the issue more simply.

Reply:
Yeah sure - that sounds saner

Reply:
Done